stem_cell/tests/test_differentiator.py

```diff
@@ -39,6 +49,12 @@ def get_version_control_token(self):
     def set_version_control_token(self, token):
         self.version_control_token = token
 
+    def set_version_control_private_key(self, private_key):
```

Comment:
In those cases maybe we should adopt @property? 

The get/set convention doesn't seem in vogue in Python: https://www.google.co.uk/?ion=1&espv=2#q=python%20get%20and%20set